### ISSUE TITLE
Highlight Dependencies

### ISSUE THREAD

**Issue by xjamundx** [TheLarkInn/js-parser-discussions#4]
Which projects are using which tool?

In my head everyone has pretty much converged around acorn, but maybe babylon will become the new standard. I have a fairly limited set of tools I work with however, so it would be nice to see a list of modern projects and which parsers they use.

eslint for example uses `espree`, but mostly as a simple translation layer between `acorn` and `esprima` for error handling and other things, since `eslint` originally used `esprima` and now uses `acorn` under the hood (via `espree`).

**Comment by TheLarkInn** [comment]
A list would be good. I agree.

**Comment by xjamundx** [comment]
Also of note are download stats for the 4 big esprima-like parsers:
1. https://www.npmjs.com/package/esprima
2. https://www.npmjs.com/package/acorn
3. https://www.npmjs.com/package/babylon
4. https://www.npmjs.com/package/espree

**Comment by ariya** [comment]
Download count can be done using shields, here is one for Esprima:  [![npm download](https://img.shields.io/npm/dm/esprima.svg)](https://www.npmjs.com/package/esprima).

However, I question the value of displaying it. For the purpose of the discussion, it can be considered as a vanity metric, a popularity contest. The merit of the discussion should be as technical as possible.

**Comment by TheLarkInn** [comment]
Agreed. I think downloads isn't needed in this case. However finding out what depends on each of them maybe has more relevance.
